lua/resession/init.lua

```diff
@@ -69,6 +69,17 @@ M.get_current = function()
   return tab_sessions[tabpage] or current_session
 end
 
+---Get information about the current session
+---@return resession.SessionInfo
+M.get_current_session_info = function()
+  local session_info = {}
+  local session = M.get_current()
+  if session_info[session] ~= nil then
+    session_info.dir = session_info[session].dir
+  end
+  return session_info
+end
```

Comment:
This will never work because the local `session_info` will override the global one. `if session_info[session]` will always be nil. Did you test this at all?

Also, you aren't setting the session name on the return value.

Reply:
Sorry, I don't know what I was thinking at the time. It should be fixed now.